webapp/src/game/game.controller.ts

```diff
@@ -56,13 +56,21 @@ export class GameController {
     type: [Game],
     description: "Get all ongoing games",
   })
-  async getOnGoingGames(): Promise<Game[]> {
+  async getOnGoingGames(
+    @Headers() headers,
+    @Res() response: Response,
+  ): Promise<any> {
     const onGoing = await this.gameService.findOnGoing();
 
     if (onGoing.length < 1) {
       throw new NotFoundException();
     }
-    return onGoing;
+
+    if (headers?.accept && headers.accept === "application/json") {
```

Comment:
You don't need the first check here, do you? If `headers.accept` is `undefined`, it won't equals `"application/json"`